### ISSUE TITLE
Is it possible to avoid cluttering the shell command history?

### ISSUE THREAD

**Issue by jbusecke** [mlshapiro/jupyterlab-flake8#14]
First of all: Many thanks for all the work on the extension. This functionality is really awesome!

I noticed that the commands running the extension in the background completely override my shell command history (I am using zsh/ohmyzsh). 
Is there any way to prevent these commands from entering the command history?

**Comment by dennisverspuij** [comment]
Experienced the same while using bash. Tried things like setting `HISTIGNORE="*flake8*"` but this doesn't work because of multiline command executed by the extension. The Jupyter terminal manager api also does not support ad hoc environment variables to be set when launching a terminal (only globally for all terminals). However, in the end I found out this simple patch should work across most shells:
```
diff --git a/src/index.ts b/src/index.ts
index 292dcc4..be22e3a 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -146,7 +146,7 @@ class Linter {
       return;
     }

-    this.term = new Terminal({initialCommand: 'echo "Opening flake8 terminal"'});
+    this.term = new Terminal({initialCommand: 'echo "HISTFILE= ; Opening flake8 terminal"'});
     try {
       this.term.session = await this.app.serviceManager.terminals.startNew();

```

**Comment by mlshapiro** [comment]
@dennisverspuij thanks for the simple solution. implemented in 0.3
